scripts/invalidate-just-cache.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs-extra');
 const path = require('path');
 
-const topLevelDep = path.join(__dirname, '../packages/prettier-rules/node_modules/.just');
+const topLevelDep = path.join(__dirname, '../packages/tslint-rules/node_modules/.just');
```

Comment:
@kenotron Will this work correctly?

Reply:
I think this just picks a random cache to make sure to invalidate cache for all. This will work okay.